### DISCUSSION TITLE
"Q&A" discussion link/button from the issue template selector

### DISCUSSION THREAD

**Discussion by nolar** [community/community#2858] [Discussions] [answered]
Can you please make it possible to add a button to the issue template selector which creates a discussion in a specific category?

It would be good if the button is big and green, same as for other issue templates. Not the black-on-black button as for the external links.

If not possible, then at least make it possible to form a link to the discussion form with a specific category pre-set. Now, the category has to be selected manually.

A use-case: I want all support questions to go to Discussions. But people are used to create issues. It will take long time until everyone learns the new feature. Having a big green button in the issue creation seems like a good idea.

Now, it looks like this ("Questions" is an issue template):

![image](https://user-images.githubusercontent.com/544296/101543278-50537a80-39a4-11eb-8bd6-68c8c6b7b12c.png)

**Comment by tomkerkhove** [comment, marked as answer]
You can easily do this by using an issue template `config.yaml`, you can find an example on https://github.com/kedacore/keda/blob/main/.github/ISSUE_TEMPLATE/config.yml#L4

**Reply by becca** [reply]
@tomkerkhove is correct that right now this is the best option.

Aside from wanting some prefilled ability/templates is there a reason you want the button in this scenario to be the same one for Discussions (in terms of look/feel) @nolar ?

**Reply by nolar** [reply]
Not necessary the same. I prefer is the buttons' look is customizable. But in the lack of button customization, the only wish is that it is not black-on-black (i.e. invisible) — people will create questions as bugs/feature requests instead.

**Reply by sagikazarmark** [reply]
While that's true, it doesn't work if you want to select the category by adding `?category=q&a`. It works for the rest of the categories.

**Comment by nolar** [comment]
@tomkerkhove Yes, but:
1. It is not a big green button, it is an invisible black button on a black background (see picture).
2. It does not lead to the Q&A category, that one has to be selected manually. — that was the second part of the request.

![image](https://user-images.githubusercontent.com/544296/101602155-f2ec1780-39fd-11eb-8c56-a82c2edc0a34.png)

**Reply by tomkerkhove** [reply]
Fair enough, at least it's something for now but dedicated URLs would be nice indeed.

**Reply by sagikazarmark** [reply]
Actually, you can add `?category=` at the end of the URL, but it doesn't work for Q&A, probably because of the ampersand. It doesn't work with urlencoding that character either, but it works just fine with the rest of the categories (general, ideas, etc)

**Reply by BetaHuhn** [reply]
You can use `-` instead of whitespace or an ampersand, so Q&A would become Q-A:

Example: https://github.com/github/feedback/discussions/new?category=discussions-feedback

**Reply by orta** [reply]
You can click on the categories in the main discussions index to grab the identifier you need, just worked well for me.

**Comment by LlmDl** [comment]
This absolutely needs to happen, preferably with the ability to pre-fill the various discussion posts like we can with tickets.

**Reply by becca** [reply]
tracking that request over in https://github.com/github/feedback/discussions/2892
